ui/src/widgets/ui-table/UITable.vue

```diff
@@ -14,7 +14,7 @@
         class="nrdb-table"
         :mobile="isMobile"
         :class="{'nrdb-table--mobile': isMobile}"
-        :items="messages[id]?.payload" :return-object="true"
+        :items="payload || []" :return-object="true"
```

Comment:
This was throwing errors previously when pass null as payload as `v-data-table` was trying to iterate through null